.github/workflows/create-scheduled-release.yml

```
@@ -0,0 +1,78 @@
+name: "Create scheduled release"
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+    - cron: "0 0 15 * *"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  get-changed-metadata:
+    name: "📋 Get a list of changed metadata"
+    runs-on: "ubuntu-20.04"
+    timeout-minutes: 5
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+      none-found: ${{ steps.set-matrix.outputs.none-found }}
+    steps:
+      - name: "☁️ Checkout repository"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: "🔧 Setup java"
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'oracle'
+          java-version: '17'
+      - name: "🕸️ Get changed metadata matrix"
+        id: set-matrix
+        run: |
+          LATEST_TAG=$(ls -1 .git/refs/tags/ | sort -V | tail -1)
```

Fiddling with `git` internals is not great. What about something like this:
```suggestion
          LATEST_TAG=$(git describe --tags --abbrev=0)
```

---

I already tried that, but the `git` command you mentioned returns the last tag from the current branch, and since our "release tags" won't be on the master, this won't work (it will always return `0.3.9` which is currently the latest tag on master, and after we create the next release (`0.3.10`) with the tag not on master, the release after that will again see `0.3.9` as the latest tag if we use the git command from above)

---

Right, how about this?
```suggestion
          LATEST_TAG=$(git tag --list | sort -V | tail -1)
```

---

That works!